source/includes/language-compatibility-table-csharp.rst

```diff
@@ -5,17 +5,18 @@
    :class: compatibility-large no-padding
 
    * - Driver Version
-     - .NET 4.5
-     - .NET 4.6
-     - .NET 4.7
+     - .NET 5
      - .NET 4.8
-     - .NET Core 1.0
-     - .NET Core 1.1
-     - .NET Core 2.0
-     - .NET Core 2.1
-     - .NET Core 2.2
-     - .NET Core 3.0
+     - .NET 4.7
+     - .NET 4.6
+     - .NET 4.5
      - .NET Core 3.1
+     - .NET Core 3.0
+     - .NET Core 2.2
+     - .NET Core 2.1
+     - .NET Core 2.0
+     - .NET Core 1.1
+     - .NET Core 1.0
```

Comment:
This ordering is incorrect. Strangely, the plain `.NET x` < 5 should follow after `.Net Core x`. See [here](https://www.tutorialsteacher.com/csharp/csharp-version-history). For reasons in the past we've omitted the word "Framework" from those early versions. I'm ok keeping this as nobody has complained, but we should keep them in order.